packages/vite/src/node/server/middlewares/indexHtml.ts

```diff
@@ -10,24 +10,27 @@ import {
   resolveHtmlTransforms,
   traverseHtml
 } from '../../plugins/html'
-import { ViteDevServer } from '../..'
+import { ResolvedConfig, ViteDevServer } from '../..'
 import { send } from '../send'
 import { CLIENT_PUBLIC_PATH, FS_PREFIX } from '../../constants'
 import { cleanUrl, fsPathFromId } from '../../utils'
 import { assetAttrsConfig } from '../../plugins/html'
 
 export function createDevHtmlTransformFn(
   server: ViteDevServer
-): (url: string, html: string) => Promise<string> {
+): (url: string, html: string, originalUrl: string) => Promise<string> {
   const [preHooks, postHooks] = resolveHtmlTransforms(server.config.plugins)
 
-  return (url: string, html: string): Promise<string> => {
+  return (url: string, html: string, originalUrl: string): Promise<string> => {
     return applyHtmlTransforms(
       html,
       url,
       getHtmlFilename(url, server),
       [...preHooks, devHtmlHook, ...postHooks],
-      server
+      server,
+      undefined,
+      undefined,
```

Comment:
**suggestion:** passing two times `undefined` into a function looks weird
Either we should look if
- we can change the order of parameters and use optionals for them
- we should use an options object that can be destructured
- we should use `null` instead of `undefined` to make it more explicit